index.html

```diff
@@ -77,7 +69,47 @@ <h4 class="sub-title">I am a Software Developer</p>
 
       </section>
 
+      <section class="wrap">
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        
+      <header>
+        <h1 class="heading">Multi-Purpose Stories</h1>
+     </header>  
+     
+     <main>
+    <section>
+        <div>
+    <p class="text">
+        A daily selection of privately 
+         personalized reads; no accounts or 
+          sign-ups required. has been the 
+           industry's standard dummy text ever 
+           since the 1500s, when an unknown 
+           printer took a standard dummy text.
+    </p>
+    </div>
+    
+    <nav class="button">
+        <button class="button1" type="css">css</button>
+        <button class="button2" type="html">html</button>
+        <button class="button3" type="bootstrap">bootstrap</button>
+        <button class="button4" type="Ruby">Ruby</button>
```

Comment:
- [ ] For semantic correctness. HTML has the ability to express lists of things, and it helps the Google robot 🤖, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better. For this reason, consider using `li` nested with `ul` tags. So you end with something like this 👇🏽: 

![image](https://user-images.githubusercontent.com/43172164/198827727-f621e468-0cb6-4f3e-a633-ff09ae20951d.png)